docs/01-get-started/installation.mdx

```diff
@@ -1,20 +1,236 @@
 ---
 title: Installation
-description: Installing Front-Commerce
+description:
+  This guide will help you get started with Front-Commerce by installing it.
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+import ContactLink from "@site/src/components/ContactLink";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-Add installation instructions here:
+## Preparation
 
-- ask for an access
-- `npx create-remix my-app --template https://example.com/remix-template.tar.gz`
-- start
-- screenshot!
+This guide will walk you through the steps to configure your system for
+installing Front-Commerce packages from our private npm package repository using
+a global `.npmrc` file.
 
-https://docs.astro.build/en/install/auto/
+### Prerequisites
+
+:::caution
+
+Front-Commerce is a **closed source product**. This documentation supposes that
+you have access to our private repositories.
 
 :::
+
+Access is granted to teams that have signed the relevant legal contracts, either
+as a **Partner** or a **Customer**. Please <ContactLink /> if you need help or
+further information.
+
+Before you begin, ensure that you have the following:
+
+- A [Blackswift Gitlab](https://gitlab.blackswift.cloud/) account authorized to
+  access our
+  [Front-Commerce private workspace](https://gitlab.blackswift.cloud/front-commerce/)
+- A
+  [Blackswift Gitlab Personal Access Token](https://gitlab.blackswift.cloud/-/profile/personal_access_tokens)
+  with the `read_api` scope
+- Node.js and npm installed on your machine
+
+### Step 1: Create a Blackswift Gitlab Personal Access Token
+
+1. Log in to your Blackswift Gitlab account.
+2. Navigate to your **Profile** and click on **Access Tokens**.
+3. Click on the
+   **[Add new token](https://gitlab.blackswift.cloud/-/profile/personal_access_tokens)**
+   button.
+4. Enter a name for your token and select the `read_api` scope.
+5. Click on the **Create personal access token** button.
+6. Copy the token value as you will need it in the next step.
+
+### Step 2: Create a `.npmrc` file
```

Comment:
**thought:** This whole sections assumes the user is on a UNIX system (notably with the `~/.npmrc`.

**thought(alt?):** Could we just redirect the user to an external documentation to _create_ the .npmrc file, and just tell what to put in ours?

Reply:
Good catch. I reworded this section with a link to https://docs.npmjs.com/cli/v9/configuring-npm/npmrc#files and mentioned `~/.npmrc` in the codeblock only.

![2023-10-05_19-00](https://github.com/front-commerce/developers.front-commerce.com/assets/75968/12cb7b1d-475f-4055-8e98-68d44e00ff1b)